automan/edm_cluster_manager.py

```diff
@@ -0,0 +1,95 @@
+import os
+from textwrap import dedent
+
+from .cluster_manager import ClusterManager
+
+
+class EdmClusterManager(ClusterManager):
+
+    # The path to edm root on the remote, this is a relative path
+    # and is relative to the home directory.
+    EDM_ROOT = '.edm'
+
+    BOOTSTRAP = dedent("""\
+    #!/bin/bash
+
+    set -e
+    EDM_ROOT={edm_root}
+    ENV_FILE="{project_name}/bundled_env.json"
+
+    if hash edm 2>/dev/null; then
+        EDM_EXE=edm
+    else
+        EDM_EXE=~/$EDM_ROOT/bin/edm
+    fi
+
+    if ! $EDM_EXE envs exists {project_name} ; then
+        if [ -f $ENV_FILE ] ; then
+            $EDM_EXE -q envs import --force {project_name} -f $ENV_FILE
+        else
+            $EDM_EXE -q envs create --force {project_name} --version 3.6
+            $EDM_EXE -q install psutil execnet -y -e {project_name}
+        fi
+    fi
+
+    source ~/$EDM_ROOT/bin/edm-activate {project_name}
```

Comment:
This will not work on systems where the edm is installed system-wide.  Instead of activating at all use this:
```
$EDM_EXE run -e {project_name} -- pip install automan
```
etc.